src/System.Configuration.ConfigurationManager/src/ApiCompatBaseline.netfx.txt

```diff
@@ -0,0 +1,3 @@
+Compat issues with assembly System.Configuration.ConfigurationManager:
+InterfacesShouldHaveSameMembers : Implementation interface member 'System.Configuration.Internal.IInternalConfigHost.GetRestrictedPermissions(System.Configuration.Internal.IInternalConfigRecord, System.Security.PermissionSet, System.Boolean)' is not in the contract.
```

Comment:
@JeremyKuhne @safern while looking at something else I noticed this. Given this member is an interface member it will make this IIternalConfigHost interface non-exchangeable with .NET Framework. What would it take to fix this so this interface matches desktop?

Reply:
Presumably this is a dangler from when we didn't have System.Security. If all we want is the interface (e.g. doesn't work) I assume it is straight forward.

Reply:
Yes we should add the interface method so they can at least exchange. Please file an issue to track adding this.